apps/schematic/notebook/prepare-python.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+pyenv install --skip-existing 3.9.2
+pyenv local 3.9.2
+poetry env use 3.9.2
```

Comment:
I noticed that for the `prepare-python` script in schematic, we are doing `pyenv local 3.10.0`. Should python version that we use for notebook be consistent with the code we use for schematic above?

Reply:
No, projects may use different versions of Python. This is actually a feature of the monorepo. However, as you suggest, it would be a good practice for the projects of the same *product* (e.g. OpenChallenges, Schematic, etc.) to use the same version of Python.